packages/notifi-frontend-client/lib/client/NotifiSubscriptionClient.ts

```diff
@@ -0,0 +1,30 @@
+import { createClient, Client } from 'graphql-ws';
+
+export const NotifiSubscriptionClient = (url: string, jwt: string | undefined, callback = () => { }) => {
+
+  const client: Client = createClient({
+    url: url,
+    connectionParams: {
+      Authorization: `Bearer ${jwt}`,
+    },
+  });
+
+  client.on('connected', () => {
+    console.log('WebSocket connection opened');
+  });
+
+  client.on('error', (error) => {
+    console.error('WebSocket error:', error);
+  });
+
```

Comment:
you need handler for:
 client.on('close', () => {
  });
  
  in this, you should check to see if we're still authenticated, and if so, reconnect after 5 seconds